README.md

```diff
@@ -3,7 +3,9 @@
 This repository contains code used in the interview process for developers joining the Microsoft Engineering Practice at ClearPoint.
 
 The application is a simple to-do list that allows a user to create items in a list and mark them as complete.
-It is a React-based front end application that uses a Microsoft Web API at the backend to facilitate using and persisting items in this to-do list.
+The front end application uses a Microsoft Web API at the backend to facilitate using and persisting items in this to-do list.
+
+The repository includes React-based **Frontend** as well as Angular-based **Frontend-angular** front end application. You can choose any one of the framework to implement your changes.
```

Comment:
The repository includes a React-based **Frontend** as well as an Angular-based **Frontend-angular** front end application. You can choose either of these frameworks to implement your changes.

Reply:
@morne-niemand Done.